modules/ROOT/pages/maven-preferences-reference.adoc

```diff
@@ -25,12 +35,12 @@ You can only edit this preference when you disable *Use embedded Maven installat
 | Empty
 
 | Override settings security path
-| When selected, this option overrides Studio default settings_security.xml path. +
+| When selected, this option overrides Maven default settings_security.xml path. +
 Selecting this option requires you to define your custom path to your settings_security.xml file in the *Path to settings_security.xml* field.
 | Disabled
 
 | Override Maven local repository location
-| When selected, this option overrides Studio default local repository location. +
+| When selected, this option overrides Maven default local repository location. +
```

Comment:
```suggestion
| When selected, this option overrides Maven's default local repository location. +
```

Reply:
According to our style guide, we should not make proper names (such as Maven) possessive.